static/js/protein/data_processing_utils.ts

```diff
@@ -287,6 +287,34 @@ export function getVarGeneAssoc(data: GraphNodes): ProteinVarType[] {
               }
               if (n2.nodes[0].value) {
                 tissue = n2.nodes[0].value;
+        */
+        if (n.property !== "geneSymbol") {
+          continue;
+        }
+        for (const n1 of n.nodes) {
+          for (const n2 of n1.neighbors) {
+            if (n1.neighbors.length !== 4) {
+              continue;
+            }
+            console.log(n1.neighbors);
+            if (n2.property === "log2AllelicFoldChange") {
+              if (_.isEmpty(n2.nodes[0].value)) {
```

Comment:
should check that `n2.nodes` is not an empty list before accessing `n2.nodes[0].value` & same for the rest of the code. You can actually do the check before line 300 because it looks like the code following line 300 requires `n2.nodes` to not be an empty list.

Reply:
Added the check !